changelogs/unreleased/8487-Lyndon-Li

```diff
@@ -0,0 +1 @@
+Fix issue #8433, add aks label to data mover pods
```

Comment:
```suggestion
Fix issue #8433, add aks label to data mover pods when node-agent has the label for workload identity.
```

Reply:
Done, changed to `add third party labels to data mover pods when the same labels exist in node-agent pods`